JenkinsfileRT

```diff
@@ -53,7 +53,7 @@ def pip_install_args = "--index-url ${pip_index} --progress-bar=off"
 env_vars = [
     "TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory",
     "CRDS_CONTEXT=roman_0051.pmap",
-    "CRDS_SERVER_URL=https://serverless",
+    "CRDS_SERVER_URL=https://roman-serverless",
```

Comment:
This change, from the crds perspective, is a NOOP. The only condition to forcing serverless mode is having "serverless" in the url, as it already was pre-change.

Reply:
Thanks for taking a look. Yes this is still an issue. Every run without this change is silently failing the crds context lookup step with the following error (which @nden pointed out):
```
++ crds list --contexts roman_0051.pmap --mappings
++ grep pmap
CRDS - ERROR -  (FATAL) CRDS server connection and cache load FAILED.  Cannot continue.
 See https://hst-crds.stsci.edu/docs/cmdline_bestrefs/ or https://jwst-crds.stsci.edu/docs/cmdline_bestrefs/
 for more information on configuring CRDS,  particularly CRDS_PATH and CRDS_SERVER_URL. : [Errno 2] No such file or directory: '/grp/crds/roman/test/config/jwst/server_config'
+ echo 'CRDS_CONTEXT = '
CRDS_CONTEXT =
```

As @ddavis-stsci commented it might be cleaner to just remove `crds list` entirely (I don't know why this was added in the first place).

I understand that both `serveless` and `roman-serverless` are equivalent for forcing serverless mode. However this change does effect what observatory is inferred from the `CRDS_SERVER_URL` as mentioned in the comment above and as seen in the jenkins runs.

Reply:
Nope, my mistake. I had presumed that the "serverless" indicator precluded any other server-based checking, but it does not. If the observatory is unknown, crds still attempts to pull the name from the CRDS_SERVER_URL. So, yes, this change will enforce the use of the roman side of crds.

So, this PR is approved.

However, curiosity question: At this point in the CI, how has `crds` been installed? Directly or presumed installed with `romancal`?

For jwst, I believe the `crds list` command was there just to log what the state of the crds system is during CI. Leaving it in or out is up to the user.

Reply:
crds is installed just prior to the call to `crds list`. It's installed as a dependency of `romancal`:
https://github.com/spacetelescope/romancal/blob/c3a7025cb92107c29c9ff5a74e43187b26b0f0b6/JenkinsfileRT#L89